packages/core/admin/admin/src/content-manager/components/DynamicZone/components/DynamicComponent.js

```diff
@@ -148,7 +148,7 @@ export const DynamicComponent = ({
             <Menu.SubTrigger>
               {formatMessage({
                 id: getTrad('components.DynamicZone.add-item-above'),
```

Comment:
Should we add translation entries for these keys?

Reply:
We could, I don't think it's that important unless we start adding translations 👍🏼